packages/node_modules/@ciscospark/widget-meet/src/messages.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * WidgetMeet Messages
+ *
+ * This contains all the text for the FeaturePage component.
+ */
+import {defineMessages} from 'react-intl';
+
+export default defineMessages({
```

Comment:
I'm still conflicted here. I inherently don't like having to include `react-intl` into what should be small self contained react components... it just ads a huge amount of overhead. I'm not sure how to resolve that.

The approach here is that we have a generic `button-control` component that just takes a `button` array as props to build up the presentational elements. That's why the i18n text is here in the container.